src/main/routes/watch.ts

```diff
@@ -17,19 +17,32 @@ export default function (app: Application): void {
         return;
       }
 
+      const recordingPlaybackDataUrl = `/watch/${req.params.id}/playback`;
```

Comment:
can you sanitise req.params.id please to avoid any injection there. Needs to be above line 12 as it's used there too. just need to ensure it's a UUID i think